dynalab/handler/handler.py.template

```diff
@@ -92,20 +92,22 @@ class Handler(BaseDynaHandler):
         response should be a single element list of a json
         the response format will need to pass the validation in
         ```
-        dynalab.tasks.{your_task}.TaskIO().verify_response(response, data)
+        dynalab.tasks.TaskIO("{your_task}").verify_response(response, data)
         ```
         """
-        response = dict()
+        response, model_response = dict(), dict()
+        answer, conf = inference_output
         example = self._read_data(data)
         # ############TODO 4: postprocess response ########
         """
         Add attributes to response
         """
         response["id"] = example["uid"]
-        response["label"] = inference_output > 0.5
-        response["prob"] = inference_output
+        model_response["answer"] = answer if answer != '[CLS]' else ''
+        model_response["conf"] = conf
+        response["model_response"] = model_response
```

Comment:
Can we put this outside the `TODO` block? It's best if users don't need to worry about the outter IO.

Reply:
Good point, will do that. On similiar lines, should we also put `response["id"] = example["uid"]` outside the TODO block? This is something that users shouldn't need to change.

Reply:
yeah if `uid` and `id` are reserved fields then we can put them outside as well.